Sources/Data Model/ProjectConfig.swift

```diff
@@ -122,6 +124,41 @@ class ProjectConfig {
             project.rollouts.forEach { map[$0.id] = $0 }
             return map
         }()
+        
+        self.flagRulesMap = {
+            var map = [String: [Experiment]]()
+            
+            project.featureFlags.forEach { flag in
+                var experiments = flag.experimentIds.compactMap { expId in
+                    return allExperiments.filter { $0.id == expId }.first
```

Comment:
do we have any `map` of experiments by ids, from where we can lookup. 
filtering every time for each experimentId may be a bit performance issue.